examples/sveltekit-server/package.json

```diff
@@ -0,0 +1,30 @@
+{
+	"name": "sveltekit-server",
+	"version": "0.0.1",
+	"private": true,
+	"scripts": {
+		"dev": "vite dev",
+		"build": "vite build",
+		"preview": "vite preview --host 0.0.0.0 --port 3000",
+		"lint": "prettier --plugin-search-dir . --check . && eslint .",
+		"format": "prettier --plugin-search-dir . --write ."
+	},
+	"dependencies": {
+		"@sveltejs/adapter-node": "^1.3.1",
+		"@sveltejs/kit": "^1.20.4",
+		"@xenova/transformers": "^2.5.2",
```

Comment:
There should be a clear split between normal dependencies and dev dependencies. What's the default for sveltekit projects? @xenova/transformers should be normal dependency, while most of the others should be dev dependencies.

Reply:
e.g., should be more like https://github.com/xenova/transformers.js/blob/1a4fe2eb70abd1fee78853602f4b9b78a62f354d/examples/sveltekit-static/package.json